### PR TITLE
Add Home Assistant to IOT category

### PR DESCRIPTION
**Pull request by stephenjust**
Home Assistant is an open-source and self-hosted home automation platform. Its interface supports TOTP as a multi-factor authentication method.

### CODE REVIEW

**Comment by stephenjust** [comment]
I'm not 100% certain how this should be listed since home-assistant.io itself is not something you sign in to. However, this isn't the only such listing on the site. For example, MyChart (under the Health category) is similar in that organizations self-host the software on their own servers / domain.

**Comment by Carlgo11** [comment]
@stephenjust I just want to confirm that you've read the [EXCLUSION.md](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md#self-hosted-services) file before I review your PR.

**Comment by stephenjust** [comment]
I can confirm that I've read it. I would argue that:
a) this meets the public interest bar since it would impact a consumer's choice in home automation system,
b) you can buy a pre-configured appliance with the software / connect to it over the internet (easily) with the premium service subscription so it doesnt have the same technical bar as hosting your own website,
c) it maintains it's own authentication database, and
d) even though MFA is delivered as a plugin, it is shipped and enabled by default.

File: img/h/home-assistant.io.svg [hunk 1]
```diff
@@ -0,0 +1 @@
... 1 0-30 0 15 15 0 0 0 22 13l33 34h-18a15 15 0 0 0-14-10 15 15 0 0 0 0 30 15 15 0 0 0 14-10h28l28 28h14l29-29h27a15 15 0 0 0 14 10 15 15 0 0 0 0-30 15 15 0 0 0-14 10h-31l-27 27v-33l32-32h52a15 15 0 0 0 14 10 15 15 0 0 0 15-15 15 15 0 0 0-29-5h-42l30-30v-25l21-22a15 15 0 0 0 7 2 15 15 0 1 0-14-9l-14 15v-46a15 15 0 0 0 10-15 15 15 0 0 0-15-15 15 15 0 0 0-15 15 15 15 0 0 0 10 15v46l-15-15zM121 324a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm44-45a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-52-7a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-5-50a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm125 7a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm68 46a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-26 48a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm28-143a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-33-40a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-40-24a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-64 0a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-52 23a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm38 37a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm85 3a6 6 0 1 1 0-12 6 6 0 0 1 0 12z" id="a"/><mask id="b" fill="#fff"><use xlink:href="#a" width="100%" height="100%"/></mask></defs><path d="M0 0h500v500H0Z" fill="#41bdf5" style="fill-rule:evenodd"/><g mask="url(#b)" fill="#fff" transform="translate(52 70)" style="fill-rule:evenodd"><path d="M0 0h396v351H0Z"/></g></svg>
```
Comment (Carlgo11) [inline img/h/home-assistant.io.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 516 516" style="fill:#fff;background:#41bdf5"><path d="M104 426V292H69a9 9 0 0 1-6-16L250 86c4-5 12-5 16 0l98 99v-19c0-4 3-7 8-7h32c4 0 8 3 8 7v67l41 43c6 6 2 16-6 16h-35v134c0 4-4 7-8 7H112c-5 0-8-3-8-7zm206-164 2-6c0-15-20-21-28-8s6 28 19 22l22 21v21l-27 27v-20a15 15 0 1 0-10 0v30l-25 25V227l20-21 7 2a15 15 0 0 0 8-28c-13-8-28 6-22 19l-18 19-19-19 2-6c0-15-20-21-28-8s6 28 19 21l21 21v105l-36-36v-29c6-2 10-8 10-14a15 15 0 1 0-20 14v19l-28-28v-28a15 15 0 1 0-10 0v32l31 31h-18a15 15 0 0 0-29 5 15 15 0 0 0 29 5h28l43 43v75l-23-23v-29a15 15 0 1 0-10 0v19l-33-33 1-7c0-15-20-20-28-8-8 13 6 28 20 22l33 33h-18a15 15 0 0 0-29 5 15 15 0 0 0 29 5h28l28 28h14l29-29h27a15 15 0 0 0 29-5 15 15 0 0 0-29-5h-31l-27 27v-33l32-32h52a15 15 0 0 0 29-5 15 15 0 0 0-29-5h-42l30-30v-25l21-21 7 1a15 15 0 0 0 8-28c-13-8-28 6-22 20l-14 14v-46c6-2 10-8 10-14a15 15 0 1 0-20 14v46zM181 406a6 6 0 1 1 0-12 6 6 0 1 1 0 12zm44-45a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-52-7a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-5-50a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm125 7a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm68 46a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-26 48a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm28-143a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-33-39a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm-40-24a6 6 0 1 1 0-12 6 6 0 1 1 0 12zm-64 0a6 6 0 1 1 0-12 6 6 0 1 1 0 12zm-52 23a6 6 0 1 1 0-12 6 6 0 1 1 0 12zm38 37a6 6 0 1 1 0-12 6 6 0 0 1 0 12zm85 3a6 6 0 1 1 0-12 6 6 0 0 1 0 12z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Those are good arguments and you've sold me on it!
Just need to minify the SVG and then I'm satisfied with the PR.

**Review by Carlgo11 — APPROVED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**